### PR TITLE
Issue 39 - Adding whatsapp plugin

### LINKED ISSUE #39: Add Whatsapp buttom
We need to add a whatsapp button to social.like.
A lot of websites is using, with better results than twitter and g+ in mobile

### PR DESCRIPTION
**Pull request by rodfersou**
closes #39

### CODE REVIEW

File: sc/social/like/browser/stylesheets/social_like.css [hunk 1]
```diff
@@ -1,6 +1,47 @@
-#viewlet-social-like>div {float: left;margin-right: 2em;}
```
Comment (rodfersou) [inline sc/social/like/browser/stylesheets/social_like.css]: Using csscomb to make css cleaner to read

**Comment by hvelarde** [comment]
@keul can you help us testing this?

**Comment by hvelarde** [comment]
@rodfersou talked to @agnogueira and he told me is better to just display this plugin in case of mobile browser as you have mentioned.

I found this: http://detectmobilebrowsers.com/

File: sc/social/like/plugins/whatsapp/browser.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+class PluginView(BrowserView):
+
+    typebutton = ''
+    gp_enabled = True
+    language = 'en'
+
+    metadata = ViewPageTemplateFile("templates/metadata.pt")
+    plugin = ViewPageTemplateFile("templates/plugin.pt")
+
+    def __init__(self, context, request):
```
Comment (hvelarde) [inline sc/social/like/plugins/whatsapp/browser.py]: I think using plone.api will reduce the number of lines of code here

File: sc/social/like/plugins/whatsapp/templates/plugin.pt [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<tal:wa condition="view/is_mobile_browser">
+    <!-- WhatsApp -->
+    <a class="whatsapp"
+       tal:attributes="href view/whatsappurl"
+       >Share in WhatsApp</a>
```
Comment (hvelarde) [inline sc/social/like/plugins/whatsapp/templates/plugin.pt]: @rodfersou this has to be translated

**Pull request merged**